.circleci/config.yml

```diff
@@ -100,6 +100,10 @@ commands:
       - run: git submodule sync
       - run: git submodule update --init
 
+  use-npm-v9:
+    steps:
+      - run: npm i -g npm@9.4.2
```

Comment:
**nit question:** would this be better?
```
- run: npm i -g npm@^9.4.2
```
I can't remember the exact syntax but the idea is to get whatever newer patch version is compared to `9.4.2` so we're not specifically pinned to `9.4.2`.

Reply:
I believe different npm  version would cause some issues, but i think its more stable to use a specific npm version.
but [the issue here](https://app.circleci.com/pipelines/github/determined-ai/determined/33313/workflows/02b1d97f-1dd9-4273-aaca-55ef66aa95f1/jobs/1169762) is that `- run: npm i -g npm` doesnt work on circleci due to the permission error. not sure how to resolve the issue